rclrs/Cargo.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Esteve Fernandez <esteve@apache.org>", "Nikolai Morin <nnmmgit@gmail
 edition = "2021"
 license = "Apache-2.0"
 description = "A ROS 2 client library for developing robotics applications in Rust"
-rust-version = "1.63"
+rust-version = "1.70"
```

Comment:
We actually depend on u128 FFI functions (and suppress warnings currently), so it'd be nice if our minimum was actually 1.78 so we could remove that.

https://github.com/ros2-rust/ros2_rust/issues/398

Reply:
I've bumped the minimum version to 1.78 because I agree that it's good to declare that the minimum version we support is the one where u128 is ABI safe.

But sadly that's not enough to let us remove the lint because [the lint is still being applied for the foreseeable future](https://github.com/rust-lang/lang-team/issues/255), I guess because it's still not guaranteed to be ABI safe across all build targets.